apps/dashboard/src/components/conditions-editor/operator-selector.tsx

```diff
@@ -0,0 +1,19 @@
+import { OperatorSelectorProps } from 'react-querybuilder';
+
+import { Select, SelectContent, SelectTrigger, SelectValue } from '@/components/primitives/select';
+import { cn } from '@/utils/ui';
+import { toSelectOptions } from '@/components/conditions-editor/select-option-utils';
+
+export const OperatorSelector = ({ disabled, value, options, handleOnChange }: OperatorSelectorProps) => {
+  return (
+    <Select onValueChange={handleOnChange} disabled={disabled} value={value}>
+      <SelectTrigger
+        size="2xs"
+        className={cn('w-18 bg-background hover:bg-bg-weak hover:text-text-strong gap-1 text-xs')}
```

Comment:
We also have the `text-label-xs`
![CleanShot 2025-01-14 at 16 29 35@2x](https://github.com/user-attachments/assets/3123d53e-99ae-41db-82ea-a94014b371d8)

this will handle things like leading, etc...